assets/js/components/KeyMetrics/ChipTabGroup/Chip.js

```diff
@@ -36,15 +40,33 @@ export default function Chip( {
 	onClick,
 	selectedCount = 0,
 } ) {
+	function Icon() {
+		return null;
+	}
```

Comment:
Let's use the `Null` component instead of defining a new component here. https://github.com/google/site-kit-wp/blob/5f9e91386e5d67de49668ba08d8e2a56f24bb6d0/assets/js/components/Null.js#L19-L21

Reply:
This has been addressed in recent commits, thanks @eugene-manuilov